Makefile

```diff
@@ -91,11 +91,18 @@ release-manifests:
 	kubectl kustomize manifests/overlays/release > _output/components.yaml
 	kubectl kustomize manifests/overlays/release-ha > _output/high-availability.yaml
 
-# Unit tests
+
+# fuzz tests
 # ----------
 
+.PHONY: test-fuzz
+test-fuzz:
+	GO111MODULE=on GOARCH=$(ARCH) go test --test.short -race -fuzz=Fuzz -fuzztime 60s ./pkg/scraper/client/resource/
```

Comment:
Because cannot use -fuzz flag with multiple packages, so here is the specific package